RxBlocking/BlockingObservable+Operators.swift

```diff
@@ -10,14 +10,20 @@
     import RxSwift
 #endif
 
```

Comment:
I actually added this in the documentation: https://github.com/ReactiveX/RxSwift/pull/1383/files/d416ff5e5c4f461c8afe262824f7d34f73d06e81#diff-6eb6c44aac5cc628cd1c254752e52e36R101

Did you want to see something besides that?